src/librustc_resolve/imports.rs

```diff
@@ -1393,8 +1393,8 @@ impl<'a, 'b> ImportResolver<'a, 'b> {
             let is_good_import =
                 binding.is_import() && !binding.is_ambiguity() && !ident.span.from_expansion();
             if is_good_import || binding.is_macro_def() {
-                let res = binding.res();
-                if res != Res::Err {
```

Comment:
Why did `res != Res::Err` stop working?
Both `Res` and `LocalDefId` implement `PartialEq`.

Reply:
Yeah but Res is redefined locally at the beginning of the file: `type Res = def::Res<NodeId>;
`. I could use `def::Res::Err` directly instead if you think its better?

Reply:
Ah, I see. `def::Res::Err` would be better.
(And `fn is_err` can be removed.)

Reply:
Fixed